api/services/SiteCreator.js

```diff
@@ -111,7 +113,9 @@ function buildInfrastructure(params, s3ServiceName) {
   return apiClient.createSiteBucket(
     s3ServiceName,
     config.env.cfSpaceGuid,
-    config.app.s3ServicePlanId
+    config.app.s3ServicePlanId,
+    null,
```

Comment:
I think function arguments need to be `undefined` and not null in order to have the default argument value be used . Would it make more sense putting the feature flag in the `createSiteBucket` function?

Reply:
Good call on `undefined`. I can move the check to the `createSiteBucket` function also